spring-cloud-gcp-docs/src/main/asciidoc/sql.adoc

```diff
@@ -56,22 +60,14 @@ public List<Map<String, Object>> listUsers() {
 | `spring.cloud.gcp.sql.credentials-location` | File system path to the Google OAuth2 credentials
```

Comment:
Is it supposed to be  `credentials.location` now?

Reply:
Yep, I'll need to add docs on this today.